public/manifest.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
@biafra23 I have been following the guidances from "Developers Google" docs but it's not quite there yet